src/api/app/assets/stylesheets/webui2/interconnect.scss

```diff
@@ -0,0 +1,7 @@
+.add_interconnect{
+  padding-left: 1.344rem;
+  padding-right: 1.344rem;
```

Comment:
but what @DavidKang wanted to do is that all the buttons have the same size (_Connect_ and _Add_) and that you can not do with the spacing classes. It would be either a bit too big or too small, which would look really weird. I think it is fine to leave it without the pading. They don't have the same size, but it LGTM:

![image](https://user-images.githubusercontent.com/16052290/53808423-738d0280-3f52-11e9-967a-4de348696b98.png)

Reply:
Fine for me